keyboards/cyboard/dactyl/config.h

```diff
@@ -0,0 +1,30 @@
+/* Copyright 2023 Cyboard LLC (@Cyboard-DigitalTailor)
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ */
+
+#pragma once
+
+/* Split keyboard settings */
+#define SERIAL_USART_TX_PIN GP0
+
+/* RGB settings */
+#ifdef RGB_MATRIX_ENABLE
+#    define WS2812_PIO_USE_PIO1 // Force the usage of PIO1 peripheral, by default the WS2812 implementation uses the PIO0 peripheral
+#    define SPLIT_TRANSPORT_MIRROR
+#    define RGB_MATRIX_LED_COUNT 80
+#    define RGB_MATRIX_DEFAULT_MODE RGB_MATRIX_SOLID_REACTIVE // Sets the default mode, if none has been set
+#    define RGB_MATRIX_DEFAULT_HUE 33 // Sets the default hue value, if none has been set
+#    define RGB_MATRIX_DEFAULT_SAT 255 // Sets the default saturation value, if none has been set
+#    define RGB_MATRIX_DEFAULT_SPD 50
+#    define RGB_MATRIX_DEFAULT_VAL RGB_MATRIX_MAXIMUM_BRIGHTNESS
+#    define RGB_DISABLE_WHEN_USB_SUSPENDED
+#    define RGB_MATRIX_KEYPRESSES
+#    define RGB_MATRIX_FRAMEBUFFER_EFFECTS
+#endif
```

Comment:
Also, A lot of this can be moved into the info.json, at this point. 

```suggestion
#define WS2812_PIO_USE_PIO1 // Force the usage of PIO1 peripheral, by default the WS2812 implementation uses the PIO0 peripheral
#define SPLIT_TRANSPORT_MIRROR
#define RGB_MATRIX_LED_COUNT 80
#define RGB_MATRIX_DEFAULT_MODE RGB_MATRIX_SOLID_REACTIVE // Sets the default mode, if none has been set
#define RGB_MATRIX_DEFAULT_HUE 33 // Sets the default hue value, if none has been set
#define RGB_MATRIX_DEFAULT_SAT 255 // Sets the default saturation value, if none has been set
#define RGB_MATRIX_DEFAULT_SPD 50
#define RGB_MATRIX_DEFAULT_VAL RGB_MATRIX_MAXIMUM_BRIGHTNESS
#define RGB_DISABLE_WHEN_USB_SUSPENDED
#define RGB_MATRIX_KEYPRESSES
#define RGB_MATRIX_FRAMEBUFFER_EFFECTS
```